### ISSUE TITLE
IE11 fallback and media queries css properties

### ISSUE THREAD

**Issue by kingmauri** [CodyHouse/codyhouse-framework#45]
Hey there!

I'm just evaluating your framework in combination with IE11. I think there is an issue with CSS properties when they are overwritten in media queries. You can test it easily with the button component of how it looks like in Chrome and in IE11. For example the button have different padding values.

The `padding-top` property has the variable `--btn-padding-y` which is related to `--space-xs`. 
`--space-xs` is calculated like that `calc(0.5 * var(--space-unit));` 

The `--space-unit` variable is changed by media query `@include breakpoint(md) {
      --space-unit:  1.25em;
    }`

The modified postcss plugin for the fallback seems to have some troubles to calculate and find all relations?

Thx for helping.

**Comment by sebastiano-guerriero** [comment]
Hi there! In CodyFrame, the whole spacing system can be updated by editing the value of the --space-unit variable on a :root level

```scss
@supports(--css: variables) { // control spacing responsiveness
  :root {
    @include breakpoint(md) {
      --space-unit:  1.25em;
    }
  }
}
```

However, this change is not propagated to older browsers (e.g., IE). In these browsers, users will see the initial spacing value. The postcss-css-variables plugin provides a fallback in most cases. Sometimes it's not ideal to provide a fallback at all (e.g., when dealing with the responsiveness of the spacing system, where the fallback would cause a LOT of extra code - think a new spacing value for each property where the spacing variable is used).

In conclusion, we do support older browsers, but we provide an experience that is slightly different - but still accessible.

More info on our spacing docs page:
https://codyhouse.co/ds/docs/framework/spacing

Cheers

**Comment by kingmauri** [comment]
ciao sebastiano
thx for quick answering!
i see your point. i thought it was kind of a bug or i missed something. so there will be some slightly differents between the ie11 and "newer" browsers. i can live with that... and clients also will do...i hope. :)
